manifests/init.pp

```diff
@@ -0,0 +1,22 @@
+# @summary A short summary of the purpose of this class
+#
+# A description of what this class does
+#
+# @example
+#   include pulpcore
+class pulpcore {
+
+  include selinux
+
+  class { selinux:
+    mode => 'permissive',
+  }
+
+  include ::redis
```

Comment:
I think we'll want to move this to a `redis.pp` most likely within this module just for organization purposes.